modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/core/repl/executor/ItIgnitePicocliCommandsTest.java

```diff
@@ -245,7 +246,7 @@ void nodeConfigShowSuggested(ParsedLine givenParsedLine) {
     }
 
     private void connected() {
-        session.connect(new SessionInfo(DEFAULT_REST_URL, null, null, null));
+        session.connect(new SessionInfo(DEFAULT_REST_URL, null, null, null, ConnectionStatus.OPEN));
```

Comment:
That is getting weird. Could you please get rid of a constructor in SessionInfo and create a builder?